site/app/models/User.php

```diff
@@ -388,6 +394,10 @@ public function updateUserNotificationSettings($key, $value) {
         $this->notification_settings[$key] = $value;
     }
 
+    public function getRegistrationSubsection() {
+        return $this->registration_subsection;
+    }
```

Comment:
you can get rid of this function, it's not needed (as using `@prop` annotation automatically makes available this getter field.